src/main/java/com/leverx/employeestat/rest/controller/DepartmentController.java

```diff
@@ -50,7 +50,7 @@ public DepartmentDTO postDepartment(@RequestBody @Valid DepartmentDTO department
     }
 
     @PutMapping
-    @ResponseStatus(HttpStatus.OK)
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
PUT request should return either 204 - no content (without response body) or 200 - OK